pyproject.toml

```diff
@@ -37,7 +37,7 @@ noos-pyk = "*"
 [tool.poetry.dev-dependencies]
 # Format
 isort = "*"
-black = {version = "*", allow-prereleases = true}
```

Comment:
🐼 

This will requires to relock the project dependencies to be on the safe side ; interesting `Poetry` isn't complaining here giveb the `pyproject.toml` isn't in line with its `poetry.lock`.

For info, such a behaviour will be the one expected with `pipenv` (the `Pipfile` is hashed into its underlying "lock" file).